CIME/case/case_run.py

```diff
@@ -322,9 +318,11 @@ def _post_run_check(case, lid):
         cpl_logs = [os.path.join(rundir, file_prefix + ".log." + lid)]
 
     cpl_logfile = cpl_logs[0]
-
     # find the last model.log and cpl.log
     model_logfile = os.path.join(rundir, model + ".log." + lid)
+    print(
```

Comment:
Do we want to keep this print statement? If so, maybe it should be a logger call?

Reply:
Thanks for pointing this out.  I have removed it along with another I found.